frontside/lib/routers/routers.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:beamer/beamer.dart';
+import 'package:frontside/views/about/about.dart';
+import 'package:frontside/views/home/home.dart';
+
+
+
+class RouteManager{
+  static const String homeRoute = "/";
+  static const String aboutRoute = "/about";
+
+  static final routerDelegate = BeamerDelegate(
+    locationBuilder: SimpleLocationBuilder(
+        routes: {
+
+          homeRoute: (context, state) => Home(),
+          aboutRoute: (context, state) => About(),
+        }
+
+    ),
+  );
+
+}
```

Comment:
1. Can you move this file under `lib/router.dart` instead? I don't think we need a `routers` directory? At least for now we'll have only one router.